<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterWindow</class>
 <widget class="QWidget" name="EnterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
                    QLineEdit {
                    border-radius: 10px;
                    padding: 10px;
                    }
                </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
                    QLineEdit {
                    border-radius: 10px;
                    padding: 10px;
                    }
                </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">      QPushButton {
          background-color: #8E44AD;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #E67E22;
      }
     </string>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>
      QPushButton {
          background-color: #F39C12;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #E67E22;
      }
     </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
